### ISSUE TITLE
Add widget support

### ISSUE THREAD

**Issue by Sebclem** [Jerrkawz/HAGearS3#21]
It will be great to have the possibility to add a widget were you can select some light and switch to have a quick access to it.

**Comment by AlexStapleton** [comment]
Yes this would be amazing instead of opening the app each time. Could also be a widget with a 2x2 grid and you can select the 4 switches / entities to view.
